subscribers/slack/worker/Processors/UserMessageProcessor.cs

```diff
@@ -37,6 +37,20 @@ internal class UserMessageProcessor : AbstractMessageProcessor {
                         _logger.LogDebug("Supplier not supported for {@AwsSnsMessage}.", awsSnsMessage);
                     }
                     break;
+                case "abr_failed":
+                    var definition2 = new {
+                        user = new {
+                            email_address = "",
+                            role = "",
+                            name = ""
+                        }
+                    };
+                    var message2 = JsonConvert.DeserializeAnonymousType(awsSnsMessage.Message, definition2);
+                        var slackMessage2 =
+$@"*:troll:*ABR failed*:troll:*
```

Comment:
Should we add API in this message to make it a bit clearer for team members? 
```suggestion
$@"*:troll:*ABR API failed*:troll:*
```